src/tcgwars/logic/impl/gen3/DragonFrontiers.groovy

```diff
@@ -1994,14 +1994,11 @@ public enum DragonFrontiers implements LogicCardInfo {
         pokeBody "Extra Smoke", {
           text "Any damage done to your Stage 2 Pokémon-ex by your opponent's attacks is reduced by 10 (before applying Weakness and Resistance)."
           delayedA {
-            /* ruling
-            Q. Say I have a Kingra-EX with “Extra Smoke” on the bench and a Stage 2 Pokémon-EX Active (other than Kingdra-EX). If my opponent attacks with “Swift” does it ignore “Extra Smoke” or not?
-            A. The effect of Extra Smoke is always attached to whatever Pokémon that the opponent attacks with, not on the Defending Pokémon nor even on the benched Kingdra-EX. And since Swift only ignores effects on the Defending Pokémon (not the player or Pokémon attacking with Swift), it does NOT ignore Extra Smoke and damage is reduced by -10. (Feb 22, 2007 PUI Rules Team)
-             */
-            after PROCESS_ATTACK_EFFECTS, {
+            // Extra Smoke was mistranslated; it is supposed to be an effect on your Stage 2 ex, meaning it gets applied AFTER W&R, and Swift will ignore it.
```

Comment:
```suggestion
            // ERRATA: Extra Smoke was mistranslated; it is supposed to be an effect on your Stage 2 ex, meaning it gets applied AFTER W&R, and Swift will ignore it. Thus, the compendium entry "Feb 22, 2007 PUI Rules Team" shall be deemed void.
```
It's important to have word "ERRATA" in code for these cases as they'll be moved to yaml files soon and otherwise they will be missed by an upcoming search operation. If there are more instances like this, please add a similar comment to the correct code places.